webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <base-card class="hc-connections">
+    <ds-space v-if="this.connections && this.connections.length" margin="x-small">
+      <ds-text tag="h5" color="soft">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </ds-text>
+    </ds-space>
+    <template v-else>
+      <p class="no-connections">{{ userName }} {{ $t(`profile.network.${type}Nobody`) }}</p>
+    </template>
+    <template v-if="this.connections && this.connections.length <= 7">
+      <ds-space v-for="follow in uniq(this.connections)" :key="follow.id" margin="x-small">
+        <!-- TODO: find better solution for rendering errors -->
+        <client-only>
+          <user-teaser :user="follow" />
+        </client-only>
+      </ds-space>
+      <ds-space v-if="this.allConnectionsCount - this.connections.length" margin="small">
+        <base-button
+          @click="fetchConnections"
+          :loading="this.isLoading"
+          size="small"
+          color="softer"
+        >
+          {{
+            $t('profile.network.andMore', {
+              number: this.allConnectionsCount - this.connections.length,
+            })
+          }}
+        </base-button>
+      </ds-space>
+    </template>
+    <template v-else-if="this.connections.length > 7">
+      <div class="overflow-container">
+        <ds-space
+          v-for="follow in uniq(this.filteredConnections)"
+          :key="follow.id"
+          margin="x-small"
+        >
+          <client-only>
+            <user-teaser :user="follow" />
+          </client-only>
+        </ds-space>
+      </div>
+      <ds-space margin="x-small">
```

Comment:
we are trying to refactor out all uses of `ds-space`, would you mind helping us with that?

Reply:
changed in a6484e3